### DISCUSSION TITLE
sum of invoice line net amount gives error BR-CO-10

### DISCUSSION THREAD

**Discussion by LuisBolle** [stephanstapel/ZUGFeRD-csharp#252] [Q&A] [answered]
Hi everyone!

I'm having trouble getting the sum of invoice line net amount (BT-106) to work. 
Where am I supposed to set the value for the BT-106 field? I'm currently doing it in the SetTotals Method for the lineTotalAmount parameter, but I#m getting this error. The value is in the .xml file but I don't know why the validator can't find it.

The value is also correct, it is indeed the sum of all invoice positions.

Did anyone else have this error and how was it fixed?

**Comment by stephanstapel** [comment]
Which error do you get? Can you share the xml file and the response from the validator?

**Reply by LuisBolle** [reply]
I'm actually getting two errors:
BR-CO-10 - Sum of Invoice line net amount (BT-106) = Σ Invoice line net amount (BT-131).
BR-S-08 - For each different value of VAT category rate (BT-119) where the VAT category code (BT-118) is "Standard rated", the VAT category taxable amount (BT-116) in a VAT breakdown (BG-23) shall equal the sum of Invoice line net amounts (BT-131) plus the sum of document level charge amounts (BT-99) minus the sum of document level allowance amounts (BT-92) where the VAT category code (BT-151, BT-102, BT-95) is "Standard rated" and the VAT rate (BT-152, BT-103, BT-96) equals the VAT category rate (BT-119).

It seems, that I can't upload the file, because .xml ist not supported in a reply. 

But which parameters from the InvoiceDescriptor are the ones for these values?
I think, that I'm filling the wrong ones...

**Reply by stephanstapel** [reply]
you can attach the file in zip format. Or paste it inline if you like

**Reply by LuisBolle** [reply, marked as answer]
[XRechnung 2400279.zip](https://github.com/stephanstapel/ZUGFeRD-csharp/files/15244431/XRechnung.2400279.zip)

Hi, sorry for the late reply, I was out of office for the past days.
Here is the .xml I changed some values, but it should work.
